pennylane_quantuminspire2/qi_device.py

```diff
@@ -0,0 +1,21 @@
+from typing import Any, Optional, Sequence
+
+from pennylane_qiskit import RemoteDevice
+from qiskit_quantuminspire.qi_backend import QIBackend
+from qiskit_quantuminspire.qi_provider import QIProvider
+
+
+class QI2Device(RemoteDevice):  # type: ignore[misc]
+
+    _qi_provider = QIProvider()
+
+    def __init__(self, backend: QIBackend, **kwargs: Any) -> None:
+        super().__init__(wires=backend.num_qubits, backend=backend, shots=backend.default_shots, **kwargs)
+
+    @classmethod
+    def get_backends(cls) -> Sequence[QIBackend]:
```

Comment:
shall we name this method as `backends` ? That makes it more consistent

Reply:
done